src/redux/modules/ayahs.js

```diff
@@ -33,8 +40,10 @@ export default function reducer(state = initialState, action = {}) {
         loading: true
       };
     case LOAD_SUCCESS:
+      let current = state.current === null ? action.result.result[0] : state.current;
```

Comment:
this is weird. it should be `let current = state.current ? state.current : action.result.result[0]`